python/tests/integration/manifests/rbac_namespace_scope.yaml

```diff
@@ -35,7 +35,7 @@ metadata:
   labels:
     app.kubernetes.io/instance: kat-rbac-singlenamespace
     app.kubernetes.io/managed-by: kat
-    app.kubernetes.io/name: emissary-ingress
+    app.kubernetes.io/name: emissary-chart
```

Comment:
using `emissary-chart` for all these seems weird, but otherwise seems ok

Reply:
Yeah, you're not wrong. Here was my thinking:

First, we're publishing a few different artifacts that are all in `oci://ghcr.io/emissary-ingress`. Having them all be in this registry makes sense. Calling the main Docker image within that registry `emissary` also makes sense.

So... what do we call the charts? We're going to have two: the main chart and the CRD chart. In the non-OCI world, the chart was named "emissary-ingress" and so it set this label to "emissary-ingress". In the OCI world, it really doesn't make sense to have a Docker image named `emissary` and a chart named `emissary-ingress`, though. I ended up settling on naming the main chart `emissary-chart` and the CRD chart `emissary-crd-chart` -- and that implies that this label should change.

We could also borrow from Linkerd, and have charts named `emissary-crds` and `emissary-control-plane`. I'm not sure I really like that, either, since the main Emissary chart doesn't just install the control plane...

So, yeah, how strongly do you feel about that? 🙂 It is definitely _possible_ to force these labels back to `emissary-ingress`, I just don't think it makes sense if the chart isn't named that...

Reply:
Yeah no, we've chatted about that before and that all makes sense with the chart naming. I mean specifically the

```yaml
 app.kubernetes.io/name: emissary-chart
 app.kubernetes.io/part-of: emissary-ingress
```

It feels a little weird to use these labels for helm stuff since  the deployment or other k8s resoruces aren't called `emissary-chart`. I just meant more like, should we stick to label conventions as prescribed by https://helm.sh/docs/chart_best_practices/labels/?

Reply:
D'oh! Thank you, I _completely_ misunderstood what you were getting at because I wasn't paying enough attention to the name of the label. 🤦‍♂️ Yes, we should indeed follow the Helm best practices. Lemme fix that...